pkg/yttlibrary/assert.go

```diff
@@ -25,6 +26,20 @@ var (
 
 type assertModule struct{}
 
+func (b assertModule) Equals(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 2 {
+		return starlark.None, fmt.Errorf("expected two arguments")
+	}
+
+	expected, _ := core.NewStarlarkValue(args.Index(0)).AsString()
+	actual, _ := core.NewStarlarkValue(args.Index(1)).AsString()
+	if expected != actual {
+		return starlark.None, fmt.Errorf("expected %s, but found %s", expected, actual)
+	}
+
+	return starlark.Tuple{starlark.None, starlark.None}, nil
```

Comment:
As a follow on to [this](https://github.com/vmware-tanzu/carvel-ytt/pull/555/files#r758812486) comment, this can return `starlark.None, nil`, instead of a tuple.